src/Compilers/CSharp/Test/Semantic/Semantics/PropertyFieldKeywordTests.cs

```diff
@@ -443,9 +562,10 @@ public int P
         }
     }
 }
-");
+", parseOptions: TestOptions.RegularNext.WithFeature("run-nullable-analysis", runNullableAnalysis));
             var accessorBindingData = new SourcePropertySymbolBase.AccessorBindingData();
             comp.TestOnlyCompilationData = accessorBindingData;
+            comp.VerifyDiagnostics();
```

Comment:
>.VerifyDiagnostics() [](http://example.com/codeflow?start=16&length=20)

Call this on result of ```CompileAndVerify``` below. This is applicable to other added/modified tests as well.

Reply:
#Pending

Reply:
Fixed